en/docs/design/api-policies/regular-gateway-policies/jwt-claim-based-access-validator.md

```diff
@@ -0,0 +1,29 @@
+                                                                                                                                     # JWT Claim Based Access Validator
```

Comment:
Let's fix the indentation of the header. This would cause rendering issues as it is detected as code.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7738/commits/6d86aa2705daa5f9fe1815e7f30745712681c825